sds_data_manager/lambda_code/SDSCode/rds_utils/db_connection.py

```diff
@@ -0,0 +1,115 @@
+import json
+import logging
+import sys
+
+import boto3
+import psycopg2
+from psycopg2 import Error
+
+from .db_ingest_query import DbIngestQuery
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+class DbConnection:
+    """
+    Class to represent the connection to the RDS database.
+
+    ...
+
+    Attributes
+    ----------
+    secret_name: str
+        the name of the secret in the AWS Secrets Manager
+    connection: psycopg2.connection
+        psycopg2 connection object to handle connection with
+        PostgresSQL database instance.
+    cursor: psycopg2.cursor
+        psycopg2 cursor obejct that allows code to execute PostgreSQL
+        commands in the database session.
+
+    Methods
+    -------
+    send_query(query):
+        sends query to the database.
+    close():
+        closes the database connection.
```

Comment:
:100: I'm a big fan of including the ``Attributes`` and ``Methods`` in the docstring, nice work!